### PR TITLE
Data Plus and related changes

### PR DESCRIPTION
**Pull request by zuluecho9**
Disregard. WIP. June 1 publish.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 1]
```diff
@@ -22,14 +22,17 @@ New Relic has two pricing models. Not sure which model you're on? See [Overview
 
 Our New Relic One pricing model is meant to be simple and to provide a lot of value even when you're [using New Relic for free](#free). Our pricing model bills on these factors:
 
-* Your [ingested data](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing). We bill on data ingested, no matter the type, making your billing easy to understand. 100 GBs per month is free. US$0.25 per GB ingested above that.
+* Your [ingested data](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing). We bill on data ingested, no matter the type, making your billing easy to understand. 100 GBs per month is free. The price per GB above that can vary depending on which [data plan](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) you're on.
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:25]:
```suggestion
* Your [ingested data](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing). We bill on data ingested, no matter the type, making your billing easy to understand. You'll receive 100 GBs per month for free. The price per GB above the free 100 GBs can vary depending on which [data plan](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) you're on.
```

Clarity edit.

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx [hunk 2]
```diff
@@ -1,24 +1,26 @@
 ---
-title: Switch to the new pricing model
+title: Switch to the New Relic One pricing model
 metaDescription: "How to switch to New Relic's newer pricing model."
 redirects: 
   - /docs/accounts/accounts-billing/switch-new-model
 ---
 
-This doc explains how to switch from our original pricing model to the New Relic One pricing model. [Learn more about pricing models.](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models)
+Some of our older customers are still on our **original pricing model**. Our newer pricing model, called the **New Relic One pricing model**, is much simpler and offers many benefits compared to our original pricing model ([learn more](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing)). This doc explains the options available for switching to the newer pricing model.
```
Comment (akristen) [inline src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:8]: Does original pricing model need any caps?

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
+Have a free account on the original pricing model? You currently have access to a wide range of New Relic features. We currently don't have a way to convert free, non-paying customer accounts to the new pricing model. So to get on the newer pricing model, and take advantage of the those benefits, you'll need to [sign up for a new account](https://newrelic.com/signup). 
+
+[Learn what you get for free on the new pricing model.](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#free)
 
 ## How paying accounts switch to new pricing [#transition-paid]
 
-If you're a paying account on our [original pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models), you may only switch to New Relic One pricing if one of these is true:
+If your organization is on our [original pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models) and you are paying us, you may only switch to New Relic One pricing if one of these is true:
```
Comment (akristen) [inline src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:18]: Maybe:

If your organization pays for data ingestion using our original pricing model, you're eligible to switch to our new pricing model if one of these is true: 

? Could see it being redundant though.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries.mdx [hunk 4]
```diff
@@ -8,31 +8,65 @@ redirects:
...
+    title="Data Plus"
+  >
 
-These limits are as follows:
+The [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) data plan includes these query limits: 
+
+* Inspected data points per minute: 100 billion
+* Inspected data points per 30 minutes: one trillion
+
+?? SHOULD WE MENTION SOMETHING ABOUT 24 HOUR LIMITS, AS WE SAY FOR ORIGINAL DATA PLAN BELOW? SHOULD WE MENTION SOMETHING ABOUT THE DISCREPANCY BETWEEN THE 100B AND 1T, AS THAT MATH IS WEIRD? ??
```
Comment (akristen) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries.mdx:61]: Calling this out so you know to update/remove it! We talked about this briefly in our meeting last week.

**Review by akristen — APPROVED** [review]
Made some line edits for clarity, but overall it was great. :)

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:8]: no, it doesn't. it's meant to be a bit of an informal wording. might just seem strange cause it's capped.

**Pull request merged**